src/query-builder/select-query-builder.ts

```diff
@@ -1205,8 +1208,21 @@ export class SelectQueryBuilder<DB, TB extends keyof DB, O>
    *   .union(db.selectFrom('pet').select(['id', 'name']))
    *   .orderBy('name')
    * ```
+   *
+   * You can provide a callback to get an expression builder:
+   *
+   * ```ts
+   * db.selectFrom('person')
+   *   .select(['id', 'first_name as name'])
+   *   .union((eb) => eb.parens(
+   *     db.selectFrom('pet').select(['id', 'name'])
```

Comment:
should this be `eb.selectFrom` and not `db.selectFrom`?

Reply:
Yep, good catch!